avocado/utils/nfs.py

```diff
@@ -0,0 +1,349 @@
+"""
+Basic nfs support for Linux host. It can support the remote
+nfs mount and the local nfs set up and mount.
+"""
+import re
+import os
+import logging
+import commands
+
+from . import path
+from . import process
+from . import service
+
+from utils_conn import SSHConnection
+import utils_misc
+
+
+def nfs_exported():
```

Comment:
I guess this is inherited so not sure it should be changed, but I really dislike the wording on this. `exported_nfs` would make more sense. Anyway I understand it's inherited and it can probably stay as is.